<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewEvaluateDialog</class>
 <widget class="QDialog" name="NewEvaluateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QLabel" name="label_EvaPointName">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>30</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>评估点</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_EvaPointName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_LayerRef">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>法律依据:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Condition">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_AdjustCondition">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>判断条件:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Solution">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>解决方案:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Solution">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>381</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sumit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>332</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OK">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>332</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string> 确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>332</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_LayerRef">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>57</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>(法律条例检索)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_unregiste">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>332</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>注销</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
